app/konnect/configure/servicehub/manage-services.md

```diff
@@ -1,152 +1,218 @@
-    versions.
+    A Service name can be any string containing letters, numbers, or characters;
+    for example, `service_name`, `Service Name`, or `Service-name`.
 
-5. (Optional) Enter a **Description**.
+1. (Optional) Enter a **Description**.
 
-6. Click **Create**.
+1. Click **Create**.
 
     A new Service is created and {{site.konnect_short_name}} automatically
     redirects to the Service's overview page.
 
 ### Update a Service
 
-1. From the left navigation menu, click **Services**.
-
-2. Select a Service from the list.
+1. In the ![service hub icon](/assets/images/icons/konnect/icn-servicehub.svg){:.inline .konnect-icn .no-image-expand}
+Service Hub, select a Service from the list.
 
-3. Edit the Service name and description directly on this page: click on either
+1. Edit the Service name and description directly on this page: click on either
 element to reveal a text box, enter the new text, then click outside of the text
 box to save.
 
-### Delete a Service
+### Share a Service
+1. In the ![service hub icon](/assets/images/icons/konnect/icn-servicehub.svg){:.inline .konnect-icn .no-image-expand}
+Service Hub, select a Service from the list.
+
+1. Click **Share Service**.
+
+1. Select a user or team to share the Service with.
+
+1. Select a role to grant to the user or team.
 
-1. From the left navigation menu, click **Services**.
+    You can share any Service that you have access to, with
+    the same role or lesser than what you have yourself. An organization admin
+    can share any Service with any role. For more information, see [
+    Manage Teams, Roles, and Users](/konnect/org-management/teams-roles-users/#entity-and-role-sharing).
+
+1. Click **Share Service** to save.
+
+### Delete a Service
 
-2. Select a Service from the list.
+1. In the ![service hub icon](/assets/images/icons/konnect/icn-servicehub.svg){:.inline .konnect-icn .no-image-expand}
+Service Hub, select a Service from the list.
 
-3. In the top right of the overview page, click the **Actions** menu and select
+1. In the top right of the overview page, click the **Actions** menu and select
 **Delete Service**.
 
-4. In the dialog that appears, confirm that you want to delete this service.
+1. In the dialog that appears, confirm that you want to delete this service.
 
-## Service Versions and Implementations
+## Service versions
 
-### Create a New Service Version
+Every Konnect Service version is associated with one runtime group.
+Any configurations for the Service version, such as implementations, plugins,
+and routes, will also be associated with the same runtime group.
 
-1. From the left navigation menu, click **Services**.
+If a Service has multiple Service versions, each Service version can be
+associated with a different runtime group, or with the same runtime group.
+Through its versions, a Service can made be available in multiple environments,
+simply by assigning Service versions to a different runtime groups.
```

Comment:
This implies that you can change the RG for a service version. You need to create a new service version in each RG, rather than assigning an existing service to a new RG

Reply:
Ah, I see you call this out below:

> You can't move a Service version from one runtime group to another.

Maybe `by creating Service versions in different runtime groups`?